pin-cushion.js

```diff
@@ -1,3 +1,5 @@
+import { PinCushionAboutApp } from "./about.js";
```

Comment:
Related to comment in about.js, please make this a default import

Reply:
Aye, done